### ISSUE TITLE
Dual HDMI not working in multi-screen/extended mode

### ISSUE THREAD

**Issue by xbotuk** [MindShow/USBDisplay#31]
Dual HDMI not working in multi-screen/extended mode

**Comment by SamJoex** [comment]
I have the same issue – except when trying to switch to Mirrored Mode, it repeatedly connects and then disconnects.

**Comment by Piter00** [comment]
yes I'm also looking for extended mode on MacBook, but its not possible

**Comment by YazdanshensAmin** [comment]
> Dual HDMI not working in multi-screen/extended mode

did you find any solution to that problem

**Comment by ronaldotantra** [comment]
please mention me if there any solution to extended mode in macos sonoma thanks
